ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -658,15 +668,16 @@ double UncoupledProcessesTimeLoop::computeTimeStepping(
         }
         else
         {
-            if (std::abs(dt -prev_dt) < std::numeric_limits<double>::min()
-                && _last_step_rejected)
+            if (std::abs(dt - prev_dt) < std::numeric_limits<double>::min() &&
```

Comment:
Yes, the condition is true only if ``dt == prev_dt`` and it is for the case that the time step was rejected with the same time step size.